src/Search/Search.scss

```diff
@@ -22,8 +22,9 @@
   .search-icon {
     background: white;
     padding: 1.5%;
-    width: .9em;
+    width: 1.7em;
     border-radius: 0 14px 14px 0;
-    box-shadow: 2px 2px black;
+		box-shadow: 2px 2px black;
```

Comment:
we'll have some issues with this.... I moved the search bar in a Header redo while I was working on Home.